test/e2e/kubectl/portforward.go

```diff
@@ -492,6 +565,10 @@ var _ = SIGDescribe("Kubectl Port forwarding", func() {
 		ginkgo.It("should support forwarding over websockets", func(ctx context.Context) {
 			doTestOverWebSockets(ctx, "localhost", f)
 		})
+
+		ginkgo.It("should support server never read request, and disconnects", func(ctx context.Context) {
+			doTestConnectionNeverReadRequest(ctx, "localhost", f)
```

Comment:
we need to add a skip indicating the issue and the reason, something like
```
e2eskipper.Skipf("Ref: https://issues.k8s.io/74551 dependency on the container runtime")
```

Reply:
otherwise the test will be failing

Reply:
fixed, PTAL